File: dish.rb
class Dishes
	def initialize(dish)
	@dish = dish;
	end

def dirty?
	if true
		puts "We should clean this #{@dish}!"
		wash
	else
		return "This #{@dish} is already clean."
end
end

def wash
puts "This dish is clean now!"
end
end

plate = Dishes.new("plate")
p plate.dirty?

File: dish.js
function dishes(dish) {
	this.dish = dish;

	this.dirty = function(){
		if (true) {
			console.log("Lets wash this dish");
			this.wash();
		}
		else {
			console.log("This dish is already clean.");
		}
	}

	this.wash = function() {
		console.log("This dish is clean now!");
	}
}


plate = new dishes("plate");
plate.dirty();

